### PR TITLE
Markdown adjusted.

### PR DESCRIPTION
**Pull request by amanmehta-maniac**
@moisseev, I believe markdown is good now. Let me know if there is something more to be done.

### CODE REVIEW

**Comment by amanmehta-maniac** [comment]
I have also corrected a faulty internal link. Kindly merge this PR.

**Comment by moisseev** [comment]
@amanmehta-maniac Markdown looks good, tank you.
It would be kind of you to squash commits like `updated*` into a single commit.

BTW The bullet list where you've fixed the link should be two-level. You can fix it as well if you like.

**Comment by amanmehta-maniac** [comment]
Yep, I would surely squash commits next time on. 
I don't really understand when you say 'two-level'. Can you elaborate?

**Comment by moisseev** [comment]
1st level: filled circle bullets
2nd level: unfilled circle bullets
3rd level: filled square bullets - these entries should be at the 2nd level
2018-03-05 09_44_29-rspamd project ideas - pale moon

**Comment by amanmehta-maniac** [comment]
Oh. I thought they are intentionally made level-3. Anyway, I have adjusted them to level-two now.
Does this look good now? :)

**Comment by moisseev** [comment]
LGTM

**Comment by amanmehta-maniac** [comment]
@fatalbanana, can you please review this?

File: doc/quickstart.md [hunk 1]
```diff
@@ -30,8 +30,9 @@ You should also consider to setup your own [local  DNS resolver]({{ site.baseurl
 ### TLS Setup
 
 It is strongly recommended to setup TLS for your mail system. We suggest to use certificates issued by [Let’s&nbsp; Encrypt](https://letsencrypt.org) as they are free to use and are convenient to manage. To get such a certificate for your domain you need to allow Let’s&nbsp;Encrypt to check your domain. There are many tools available for these purposes, including the official client and couple of alternative clients, for example [acmetool](https://github.com/hlandau/acme). The setup is fairly simple: just type
-	add-apt-repository ppa:hlandau/rhea
-	apt-get update
+
+    add-apt-repository ppa:hlandau/rhea
```
Comment (fatalbanana) [inline doc/quickstart.md]: Maybe it's worth qualifying this with a comment indicating for what and on which platforms it's needed.

Comment (amanmehta-maniac) [reply doc/quickstart.md]: Agreed, will do.

Comment (amanmehta-maniac) [reply doc/quickstart.md]: kindly refer to #317

**Pull request merged**